charts/argo-cd/templates/_helpers.tpl

```diff
@@ -270,3 +274,34 @@ Create the name of the configmap to use
     {{ default "argocd-notifications-cm" .Values.notifications.cm.name }}
 {{- end -}}
 {{- end -}}
+
+{{/* 
+    Create Hostname helper -- Coreweave Use Only
+*/}}
+{{- define "coreweave.externalDnsName" -}}
+{{ default (printf "argocd.%s.ord1.ingress.coreweave.cloud" .Release.Namespace) .Values.customExternalDnsName }}
+{{- end -}}
+
+{{/*
+    Create DRY Helpers
+*/}}
+{{- define "coreweave.nodeAffinityAndTolerations" -}}
+nodeSelector:
+  node.coreweave.cloud/class: cpu
```

Comment:
Don't need the selector w/ the affinity below - one or the other. It's a no-op change, but might just confuse a future dev working on this chart.

Reply:
removing it, thanks!